www/index.html

```diff
@@ -33,10 +33,11 @@
         <link rel="stylesheet" type="text/css" href="css/index.css">
         <link rel="stylesheet" type="text/css" href="css/mod.css">
         <link rel="stylesheet" type="text/css" href="css/flex-objects.css">
+        <link rel="stylesheet" type="text/css" href="css/entry.css">
+        <link rel="stylesheet" type="text/css" href="css/page.css">
 
         <!-- MDL resources -->
         <link rel="stylesheet" href="lib/mdl/material.css">
```

Comment:
We eventually need to get rid of material.css, but it affects our fonts right now.